src/NzbDrone.Core/Tv/MovieService.cs

```diff
@@ -181,6 +187,31 @@ public Movie FindByTitle(string title, int year)
 
         public void DeleteMovie(int movieId, bool deleteFiles)
         {
+            //this next block was added in order to implement listsynccleaning
+            //start of block
+            if (deleteFiles)
+            {
+                List<MovieFile> movieFilesList = _mediaFileService.GetFilesByMovie(movieId);
+                //string dirPath = null;
+                foreach (var movieFile in movieFilesList)
+                {
+                    var series = GetMovie(movieFile.MovieId);
+                    var fullPath = Path.Combine(series.Path, movieFile.RelativePath);
+                    //dirPath = series.Path;
+                    _logger.Info("Deleting episode file: {0}", fullPath);
+                    _recycleBinProvider.DeleteFile(fullPath);
+                    _mediaFileService.Delete(movieFile, DeleteMediaFileReason.NotInList);
+                    //TODO: files are being deleted, but empty directory left behind??
+                    //perhaps need to delete the series path too?
+                }
+                //if (dirPath != null)
+                //{
+                //    _logger.Info("Deleting Movie folder: {0}", dirPath);
+                //    _recycleBinProvider.DeleteFolder(dirPath);
+                //    _movieFileRepository.Delete(dirPath);                    
+                //}
+            }
+            //end of block
             var movie = _movieRepository.Get(movieId);
             _movieRepository.Delete(movieId);
             _eventAggregator.PublishEvent(new MovieDeletedEvent(movie, deleteFiles));
```

Comment:
thats basically what i had, i just changed the reason to something else other than manual (though i doubt it mattered)... also. i looped.. since you could have other movie files, like subtitles etc.... (though the loop only does one iteration if there is only 1 file).... so should i put that back in.. or will you update the eventhandler?